spec.html

```diff
@@ -27229,7 +27227,7 @@ <h1>Number.prototype.toFixed ( _fractionDigits_ )</h1>
           1. Let _f_ be ? ToInteger(_fractionDigits_).
           1. Assert: If _fractionDigits_ is *undefined*, then _f_ is 0.
           1. If _f_ &lt; 0 or _f_ &gt; 100, throw a *RangeError* exception.
-          1. If _x_ is *NaN*, return the String *"NaN"*.
+          1. If _x_ is not finite, return ! Number::toString(_x_).
```

Comment:
`±Infinity` is handled a few steps further:

```
8. If x ≥ 10^21, then
  a. Let m be ! ToString(x).
```

Reply:
if `x` can only be finite or `NaN` here, why is it clearer to use "is not finite", rather than, say, leaving the `is NaN` check but adding `Assert: _x_ is finite` after it?

Reply:
`x` can be anything here, `±Infinity` is handled 2 steps below, making `Number.prototype.toFixed` change observably equivalent.

Reply:
@ljharb I would also appreciate your feedback on https://github.com/tc39/ecma262/pull/1938#issuecomment-610051371.

Reply:
ah, i see, so this is basically "hoisting" the infinities handling up to this line, this seems good.